src/forms/FormFieldImageVideoLinkDocumentBrowsing/ImageVideoLinkDocumentBrowsingTypes.tsx

```diff
@@ -29,7 +30,7 @@ export type ImageVideoDocumentLinkBrowsingDef = {
    * It should fill the assetProperties array with the document
    * properties.
    */
-  handleSetImage: () => Promise<void>;
+  handleSetImage?: () => Promise<void>;

```

Comment:
It makes sense because we don't want to lock users to always having to upload images, but what happens if the dev doesn't pass any of the callbacks?

Reply:
If we do not define any callback the following fallback text appears:

![image](https://user-images.githubusercontent.com/87880265/215794897-bb90af93-f483-4619-a2df-b9df935475ce.png)